torch/_inductor/select_algorithm.py

```diff
@@ -147,21 +157,34 @@ def __init__(
         self.triton_meta: Optional[Dict[str, object]] = None
         # For Templated Attention this can be a list of ir.Subgraph
         self.subgraphs: Optional[List[ir.ComputedBuffer]] = subgraphs
+
+        # The following attributes (body, template_mask, output_val) are all
+        # used for triton kernel codegen.
```

Comment:
nit: maybe write a note around the named_tuple and point back there, describing what these things are used for

Reply:
Doing in a follow-up PR